### PR TITLE
Fix flow of setting system idtf for files

### PR DESCRIPTION
**Pull request by KovalM**

### CODE REVIEW

File: ui/self/ui_controls.scs [hunk 1]
```diff
@@ -87,8 +87,11 @@ ui_control_search_idtf_edit
 		... (* -> [control that allows to search sc-elements by their identifiers] (* <- lang_en;; *);; *);;
 	*);;
 
-ui_control_search_idtf_edit_example = "file://contents/ui_control_search_idtf_edit_example.gif";;
-
+image
+-> "file://contents/ui_control_search_idtf_edit_example.gif"
+	(*
+		=> nrel_system_identifier: [ui_control_search_idtf_edit_example];;
+	*);;
```
Comment (deniskoronchik) [inline ui/self/ui_controls.scs:94]:
```suggestion
"file://contents/ui_control_search_idtf_edit_example.gif"
  <- image;
  <= nrel_system_identifier: [ui_control_search_idtf_edit_example];;
```

File: ui/ui_start_help.scs [hunk 2]
```diff
@@ -1,4 +1,11 @@
-ui_start_help <- lang_ru;;
-ui_start_help = "file://ui_start_help/text.html";;
+lang_ru
+-> "file://ui_start_help/text.html"
+	(*
+		=> nrel_system_identifier: [ui_start_help];;
+	*);;
```
Comment (deniskoronchik) [inline ui/ui_start_help.scs:5]:
```suggestion
"file://ui_start_help/text.html"
  <- lang_ru;
  <= nrel_system_identifier: [ui_start_help];;
```

File: ui/ui_start_help.scs [hunk 3]
```diff
@@ -1,4 +1,11 @@
...
 
-ui_help_question_movie = "file://ui_start_help/help-question-movie.gif";;
+image
+-> "file://ui_start_help/help-question-movie.gif"
+	(*
+		=> nrel_system_identifier: [ui_help_question_movie];;
+	*);;
```
Comment (deniskoronchik) [inline ui/ui_start_help.scs:11]:
```suggestion
"file://ui_start_help/help-question-movie.gif"
  <- image;
  <= nrel_system_identifier: [ui_help_question_movie];;
```

**Review by deniskoronchik — CHANGES_REQUESTED** [review]
More readable formatting

File: ui/self/ui_controls.scs (hunk 1 above)
Comment (ShunkevichDV) [reply ui/self/ui_controls.scs:94]: Yes, the better idea it to start description form an entity being described, not the some another concept

**Review by ShunkevichDV — COMMENTED** [review]
Agree with deniskoronchik suggestion

**Review by ShunkevichDV — APPROVED** [review]

**Comment by deniskoronchik** [comment]
Fixup review fixes into previous commit?
Because it changes all lines in last one

**Comment by KovalM** [comment]
> Fixup review fixes into previous commit?
> Because it changes all lines in last one

I agree. Done.

**Review by MikhailSadovsky — APPROVED** [review]

**Review by deniskoronchik — APPROVED** [review]

**Pull request merged**